DEVELOPMENT.md

```diff
@@ -54,10 +54,32 @@ hatch run test
 ```
 
 ## Run integration tests
+- Pre-requisites:
+  - Developer must have a valid AWS account.
+  - Developer must create a Farm, Queue. This can be done easily via Deadline Cloud's AWS Console quick farm create workflow. Please copy the Farm, Queue resource IDs for the next step. Please copy the Queue's Job Attachment S3 bucket name for the next step.
+
+- Setup Environment Variables
+```bash
+export SERVICE_ACCOUNT_ID=001122334455
+export JOB_ATTACHMENTS_BUCKET="{deadline queue configured Job Attachments S3 bucket.}"
+export JA_TEST_ROOT_PREFIX="/your/path/root"
+export FARM_ID="farm-{uuid}"
+export AWS_DEFAULT_REGION="us-west-2"
+```
+
+- Optional Environment Variables
+```bash
+export AWS_ENDPOINT_URL_DEADLINE="https://deadline.{region}.amazonaws.com"
```

Comment:
```suggestion
export AWS_ENDPOINT_URL_DEADLINE="https://deadline.$AWS_DEFAULT_REGION.amazonaws.com"
```

Could do this since we require `AWS_DEFAULT_REGION` above

Reply:
Good idea!